### DISCUSSION TITLE
GitHub Marketplace Models

### DISCUSSION THREAD

**Discussion by rajyraman** [microsoft/semantic-kernel#8264] [Q&A] [answered]
Is there no connector or generic chat completion client to work with models in GitHub Marketplace? Even GitHub suggests to use Azure AI Inference SDK.

![image](https://github.com/user-attachments/assets/ec0155b0-cea3-445a-9dc5-3d66c04c68d6)

**Comment by markwallace-microsoft** [comment, marked as answer]
@rajyraman We are working on a Connector that uses the Azure AI Inference SDK. Here is the feature branch https://github.com/microsoft/semantic-kernel/tree/feature-connectors-azureaiinference. This new connector will be released in the coming weeks.
